docs/guides/systemjs-webpack.md

```diff
@@ -0,0 +1,93 @@
+# Using Enzyme with SystemJS/Webpack
+
+If you are using a test runner that runs code in a browser-based environment,
+you may be using SystemJS or Webpack in order to bundle your React code.
+
+These tools use static analysis to create a dependency graph at build-time of
+your source code to build a bundle. Enzyme has a handful of conditional
+`require()` calls in it in order to remain compatible with React 0.13 and
+React 0.14.
+
+Unfortunately, these conditional requires mean there is a bit of extra setup with bundlers like SystemJS and Webpack.
+
+In your bundler configuration, you need to make sure that the compatibility
+files not needed for your version of react are marked as "external", which
+means they will be ignored.
+
+## React 0.14 Compatibility
+
+If you are using React 0.14, your configuration might include the following:
+
+### SystemJS
+
+```json
+/* package.json */
+{
+  "jspm": {
+    "overrides": {
+      "npm:enzyme@2.3.0": {
+        "map": {
+          "cheerio": "@empty",
```

Comment:
Not super sure about this line. Perhaps @guybedford knows the correct mapping to Webpack's `'window'`?